modules/caddyhttp/celmatcher.go

```diff
@@ -42,7 +42,7 @@ func init() {
 // MatchExpression matches requests by evaluating a
 // [CEL](https://github.com/google/cel-spec) expression.
 // This enables complex logic to be expressed using a comfortable,
-// familiar syntax.
+// familiar syntax. The standard definitions of CEL functions and operators can be found [here](https://github.com/google/cel-spec/blob/master/doc/langdef.md#standard-definitions).
```

Comment:
I know it has no effect but my eyes can't help. Is that line not too long?

Reply:
The sight of it bothers me too. I think it'll be wrapped when rendered though. Putting that sentence on a whole line just by itself doesn't feel right. I don't know whether new-lines within sentence will be rendered into actual <br>. @mholt, how does your doc generator handle it?

Reply:
```suggestion
// familiar syntax. The standard definitions of CEL functions and operators can be found
// [here](https://github.com/google/cel-spec/blob/master/doc/langdef.md#standard-definitions).
```

Somewhat matches how the line above with `[CEL]` looks :shrug:

Reply:
Ha, I had the same question as @abiosoft . Do the best you can with long links, even if they get their own wrapped line, it's OK; better than the whole sentence extending waaaay out there IMO.